modules/ROOT/pages/_partials/common-note-mule4.adoc

```diff
@@ -2,4 +2,4 @@
 
 * The examples above use the File connector to read a file in the middle of the flow. The information posted through the `<http:listener>` component is written to each FTP or SFTP site multiple times. Because the component makes use of the xref:4.1@mule-runtime::streaming-about.adoc[repeatable streams feature], you do not need to worry about consuming the stream multiple times.
 
-* The `<foreach>` component automatically goes through each line of the CSV file. In Mule 3, you first needed to transform the CSV file into a Java structure, but because Mule 4 is now Java agnostic, this works out-of-the-box.
\ No newline at end of file
+* The `<foreach>` component automatically goes through each line of the CSV file. In Mule 3, you had to transform the CSV file into a Java structure, but because Mule 4 is Java agnostic, this works by default.
```

Comment:
"Java agnostic" -> "Java-agnostic" ?

", this works by default." -> "no transformation is needed" ?

Reply:
Changed sentence to "In Mule 3, you had to transform the CSV file into a Java structure, but because Mule 4 is language independent, you can transform the file without first casting the CSV content as a Java structure."